core/src/main/java/org/htsjdk/core/test/HtsjdkBaseTest.java

```diff
@@ -0,0 +1,7 @@
+package org.htsjdk.core.test;
```

Comment:
I agree to decide later how to package test-helpers, but better if at least the classes live in `org.htsjdk.test` instead of inside `core` (easier to split later and separate from the concept of `core`)

Reply:
done